buildscripts/incremental/create_conda_environment.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Setup environment
+conda update --yes conda
+# Scipy, CFFI and jinja2 are optional dependencies, but exercised in the test suite
+conda create -n travisci --yes python=$PYTHON numpy=$NUMPY cffi pip scipy jinja2
```

Comment:
yeah. Again, I have merely refactored the code. Improvements can come later. :-)